route_resources:
  - resource_id: "Resource01"
    floor_id: "27F"
    coord_x: 62.86893081665039
    coord_y: -92.57209777832031
